app/(pages)/ecommerce/_components/EcommercePricingPlansSection/EcommercePricingPeriodToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from "react";
+
+import { ToggleButton } from "components";
+
+type TProps = {
+  activePeriod: string;
+  setActivePeriod: (period: string) => void;
+};
+
+export const EcommercePricingPeriodToggle: FunctionComponent<TProps> = ({
+  activePeriod,
+  setActivePeriod,
+}) => {
+  return (
+    <div className="flex p-1 bg-black rounded-xl w-full max-w-[354px]">
```

Comment:
It shouldn't be a div, I think it should be a form with radio buttons, because you can choose a single field, and than it should fill out the form with that based on a user input, so when the user executes contact us form we should know what plan the user choose